src/main/java/Deadline.java

```diff
@@ -0,0 +1,34 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task  {

```

Comment:
Hi, i think it would be nice to add java docs for this!

Reply:
Added! Thanks!